### DISCUSSION TITLE
Should uproot with `open_files=False` and the file access report enabled fail if the first file is not accessible?

### DISCUSSION THREAD

**Discussion by ikrommyd** [scikit-hep/uproot5#1253] [General]
If the first file of your dataset is not accessible to read metadata and the uproot report is enabled, should it fail like it does now or should it try the next files and let the file access error be caught by the uproot report? I find this a common issue in datasets we querried from rucio where the first file is just not accessible and I have to shuffle the fileset before passing it to nanoevents.

**Comment by jpivarski** [comment]
As we talked about in the meeting, we could add a parameter for the number of files in the file-list to try before giving up (something like 3 by default).
